registration-processor/registration-processor-message-sender-impl/src/main/java/io/mosip/registration/processor/message/sender/service/impl/MessageNotificationServiceImpl.java

```diff
@@ -264,6 +272,49 @@ public ResponseDto sendEmailNotification(String templateTypeCode, String id, Str
 
 		return response;
 	}
+	
+	private List<String> getPreferredLanguages(String id, String process) throws ApisResourceAccessException, 
+	PacketManagerException, JsonProcessingException, IOException {
+		String preferredLang=packetManagerService.getField(id, MappingJsonConstants.PREFERRED_LANGUAGE, process, 
```

Comment:
use packetManagerService.getFieldByMappingJsonKey() here

Reply:
I have changed the code.